### DISCUSSION TITLE
[Feature Request] Rename a Namespace to another Namespace

### DISCUSSION THREAD

**Discussion by chuongmep** [dotnet/docfx#8514] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
- Problem is I have two library, I want to show it in API merge to one namespace only, how to can do that, Example : 
1. OpenMEP
2. OpenMEPSanbox
I want `OpenMEPSanbox` automatically change to `OpenMEP` and document will be show just one namespace with `OpenMEP` only.

**Describe alternatives you've considered**

**Additional context**

**Comment by yufeih** [comment]
Are you asking a way to display namespaces for types in a way that does not truthfully match its definition in code?

**Comment by chuongmep** [comment]
@yufeih yes, I mean is now I need merge two library with difference assembly to one assembly, this can be done ? Because In Dynamo, we split to two namespace for sanbox and another software, with sanbox, we will change from namespace OpenMEPSanbox to OpenMEP.

**Comment by yufeih** [comment]
Once you changed the two namespaces to be the same, the output should merge into 1 toc.
